make-money/rescue-moment.md

```diff
@@ -0,0 +1,38 @@
+# Rescue Moment
+
+Rescue saves a user in a moment of need through brand sponsored value exchange ads. Rescue requires the player to opt- in to interact with an ad in return for in app-currency or extra gameplays. Our rescue unit inventory includes rewarded video and rich media interstitials. 
```

Comment:
Extra space after 'opt-' and I think it is 'in-app currency' instead of 'in app-currency'